source/jormungandr/jormungandr/interfaces/v1/Uri.py

```diff
@@ -163,17 +165,21 @@ def get(self, region=None, lon=None, lat=None, uri=None, id=None):
 
         if not self.region:
             return {"error": "No region"}, 404
+        uris = []
         if uri:
             if uri[-1] == "/":
                 uri = uri[:-1]
             uris = uri.split("/")
             if collection is None:
                 collection = uris[-1] if len(uris) % 2 != 0 else uris[-2]
-            args["filter"] = self.get_filter(uris, args)
+        param_filter = self.get_filter(uris, args)
+        if param_filter:
+            args["filter"] = param_filter
```

Comment:
Minor: you can directly do that and remove the 2 lines above (check below shouldn't be affected)

Reply:
You're right, sorry!
Nullifying filters is bad :-/